waiter/src/waiter/state.clj

```diff
@@ -212,7 +254,7 @@
                                           expired-instance-ids))
             sorted-instance-ids (->> (concat healthy-instances unhealthy-instances expired-instances)
                                      (set)
-                                     (sort-instances expired-instance-ids)
+                                     (sort-instances-for-processing expired-instance-ids)
```

Comment:
Do we want our instance distribution to be affected by expiry?

Reply:
No and they are not currently.